catalog/opensearch/catalog-opensearch-endpoint/src/main/java/org/codice/ddf/opensearch/query/filter/BBoxSpatialFilter.java

```diff
@@ -30,15 +29,35 @@
    * @param bbox comma-delimited list of lat/lon (deg) bounding box coordinates
    *     (West,South,East,North)
    */
-  public BBoxSpatialFilter(String bbox) {
+  public BBoxSpatialFilter(final String bbox) {
     super();
 
-    String[] bboxArY = bbox.split(" |,\\p{Space}?");
+    String[] bboxArY = bbox.split(OpenSearchConstants.BBOX_DELIMITER);
```

Comment:
❓  Is there a reason the `bbox` string format was made more restrictive?

Reply:
The spec says that the bbox query parameter must be in an exact format:
https://github.com/codice/ddf/blob/3d8784a8d77590b5aaed142d80ea8a68ae4caad0/catalog/opensearch/catalog-opensearch-endpoint/src/main/resources/ddf-os.xml#L59-L60